bundle/next/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -375,36 +375,23 @@ spec:
         version: v1
   description: |
     A collaborative Kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
-    This operator installs PostgreSQL, Keycloak, Plugin registry, Devfile registry and the Eclipse Che server, as well as configures all three services.
+    This operator installs PostgreSQL, Plugin and Devfile registries, Dashboard, Gateway and the Eclipse Che server, as well as configures all these services.
+    OpenShift OAuth is used directly for authentication. TLS mode is on.
```

Comment:
> TLS mode is on

Do we still support non TLS mode? If no, maybe just drop the phrase or say that it just uses HTTPS for external routes.

Reply:
We don't support non TLS mode. I've added this phrase just to emphasize